RunningOrder/WelcomeView.swift

```diff
@@ -12,14 +12,24 @@ struct WelcomeView: View {
 
     @Binding var space: Space?
     @State private var newSpaceName = ""
+    @State private var hasErrorOnField = false
 
     var body: some View {
         VStack(alignment: .leading, spacing: 20) {
             Text("Welcome").font(.largeTitle)
             Text("You don't have yet your space, or joined a shared space")
             HStack {
                 TextField("My Space Name", text: $newSpaceName)
+                    .overlay(Rectangle()
+                                .strokeBorder(Color.red, lineWidth: 2.0, antialiased: true)
+                                .opacity(hasErrorOnField ? 1 : 0))
+                    .animation(.default)
                 Button("Create") {
+                    hasErrorOnField = newSpaceName.isEmpty
+                    guard !newSpaceName.isEmpty else {
```

Comment:
```suggestion
                    guard !hasErrorOnField else {
```

Reply:
oooooh :O I thought I wrote it like this ! :O